src/hello.js

```diff
@@ -21,6 +21,7 @@ if (FEATURE_FLAG_KEY == null || FEATURE_FLAG_KEY.length <= 0) {
 // Create Unlaunch client object by using SDK_KEY and finetune configuration if needed by passing it as argument          
 var factory = UnlaunchFactory({
     core: {
+        host: 'http://localhost:5000',
```

Comment:
I think you forgot to revert it to Unlaunch

Reply:
thanks Mahrukh

Reply:
actually it was already taken care of :) https://github.com/unlaunch/hello-node/pull/2/files

Reply:
Oh sorry Tuan, I missed to see that.